src/lwc/geFormService/geFormService.js

```diff
@@ -42,6 +43,24 @@ class GeFormService {
     fieldTargetMappings;
     donationFieldTemplateLabel;
 
+    getPaymentTransactionStatusEnums = async () => {
```

Comment:
Love the move here to `geFormService`. I'm just a little worried about the `handleError` function here, In `geFormTokenizeCard`, the `handleError` is tailored toward payment/transaction errors and is a banner inside of the widget, but if we start using this we start popping up toast error messages instead.

Also, side note, should we be looking to move this handleError function to a more generic class? We keep importing `utilTemplateBuilder` everywhere.